src/index.js

```diff
@@ -31,6 +32,21 @@ const store = configureStore(reduxState)
 
 const history = syncHistoryWithStore(browserHistory, store)
 
+// Check if token existed in localStorage and expired
+// token can be stored in localStorage in two scenario
+// 1. oAuth
+// 2. TWReporter account sign in
+const { auth } = store.getState()
+if(auth.authenticated && auth.authInfo && (auth.authType=== 'facebook' || auth.authType==='google')) {
+  setupTokenInLocalStorage(auth.authInfo)
+  store.dispatch(deletAuthInfoAction())
+}
```

Comment:
the codes and comments seems not to be consistent.

Is the business logic to set token into localStorage and delete it from redux state?
If so, why?

Reply:
Sorry for the confusion. I update the comment and please check the comment again. 

Yes, The auth info pass from API to front end app by three mediums. Cookie -> Redux State -> localStorage. So, after the info package is stored in localStorage, I suppose that it should be removed from the redux state. BTW, it is only for oAuth .